crates/networking/p2p/rlpx/eth/eth68/receipts.rs

```diff
@@ -26,22 +26,22 @@ impl Receipts68 {
             };
         }
         let transformed_receipts = receipts
-            .iter()
-            .map(|receipts| receipts.iter().map(ReceiptWithBloom::from).collect())
+            .into_iter()
+            .map(|receipts| receipts.into_iter().map(ReceiptWithBloom::from).collect())
             .collect();
         Self {
             id,
             receipts: transformed_receipts,
         }
     }
 
-    pub fn get_receipts(&self) -> Vec<Vec<Receipt>> {
+    pub fn get_receipts(&mut self) -> Vec<Vec<Receipt>> {
```

Comment:
Maybe we should change the name to `take_receipts`? as to reflect the new behaviour